src/common/constants/contributors.json

```diff
@@ -355,4 +355,24 @@
     "name": "solox",
     "contributes": ["Translation (Russian)"]
   }
-]
+],
```

Comment:
@Xurwi This bracket should be at the end closing entire array/list of contributors.